gomill_setup/setup.py

```diff
@@ -89,7 +89,6 @@ def run(self):
             if len(egg_infos) > 1:
                 self.warn("multiple .egg-info files; not removing any:\n%s"
                           % "\n".join(egg_infos))
-                egg_info_pathname = None
```

Comment:
this might warrant a quick glance that this wasn't supposed to change something else.